pom.xml

```diff
@@ -45,10 +45,10 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
     <maven-project-info-reports-plugin.version>2.9</maven-project-info-reports-plugin.version>
-    <jenkins.version>1.625.3</jenkins.version>
-    <java.level>7</java.level>
-    <jgit.version>4.5.4.201711221230-r</jgit.version>
-    <jgit.javadoc.version>4.5.0.201609210915-r</jgit.javadoc.version>
+    <jenkins.version>2.60.1</jenkins.version>
+    <java.level>8</java.level>
+    <jgit.version>4.10.0.201712302008-r</jgit.version>
+    <jgit.javadoc.version>4.10.0.201712302008-r</jgit.javadoc.version>
```

Comment:
BTW why are these even distinct?

Reply:
When the JGit project releases a new patch version (for example, 4.5.4), they sometimes fail to publish the javadoc for that specific minor version.  Instead, they rely on the javadoc for the matching minor version (for example: 4.5).

I could collapse the two properties into a single one now, and then only expand them again if the JGit project fails to publish javadoc for some future patch release.

Reply:
done